INSTRUMENTS/Stepper_Motor/STEPPERDRIVER_TIC.py

```diff
@@ -0,0 +1,51 @@
+from flojoy import flojoy, DataContainer
+from ticlib import TicUSB
+from time import sleep
+
+
+@flojoy
+def STEPPERDRIVER_TIC(dc, params):
+    tic = (
+        TicUSB()
+    )  # Declaration of the stepper driver (You can add serial number to specify the driver)
+
+    tic.halt_and_set_position(0)  # Set the position to 0
+    tic.set_current_limit(
+        30
+    )  # Set the current limit to the max the driver needs this current to work correctly
+    tic.energize()  # Turn on the driver
+    tic.exit_safe_start()  # The driver is now ready to receive commands
+    positions = [
+        int(params["position1"]),
+        int(params["position2"]),
+        int(params["position3"]),
+        int(params["position4"]),
```

Comment:
4 motor routines seems a bit arbitrary. 

Can we make this more general by passing everything as a list? 

For example.

```
params['positions'] = [23, 35, 343, 0]
params['speeds'] = [1, 2, 8, 6]
```

then you can define whatever program you want for a single stepper motor - with an arbitrary number of positions. and speeds. you could also generate the positions and speeds programmatically through prior nodes that generate arrays

we'll need a new controls UI component for this (like a widget that allows you to create ordered lists manually - https://github.com/flojoy-io/studio/issues/359), but for now, you can simply not pass at param and set these arrays as defaults at the top of the function

Reply:
Okay, this seems logical but how can I avoid having "magical numbers" if I set these arrays as default at the top of the function?